content/participants/ttjc@Megha-Pathak/index.md

```diff
@@ -39,20 +39,36 @@ I have done projects along side with learning technologies some of them are:
 
 My projects:
 
-#### Not Hacked: Card Game to educate kids about Cyber Security
+#### LadeBug: Meet your online digital Ladybug for companionship & debugging -- LadeBug!
 
-_description_ This is a card game that takes the player through different privacy situations and ask for the response they will choose. The points given to them specifies how good their choice is. 
 
-_stack_ Made with HTML, CSS and JavaScript.
+_description_ A chrome extension to overlay a tiny ladybug, as a digital companion during debugging and general development work. Follow the Set-Up Instructions present on the repository.  
 
-_github link_ https://github.com/Megha-Pathak/Not-Hacked
+_stack_ Made with JavaScript.
 
-#### Know More: A Twitter Bot
+_github link_ https://github.com/Megha-Pathak/LadeBug
 
-_description_ A twitter bot to analyze the sentiments of the tweets and compare the information present in the with reliable sources. This bot aims to segregate fake information from the real one to save fake information from being spreaded. The developement of this project is currently on going so it is not hosted yet. 
+#### Hacket - Privacy card game
 
-_stack_ Django, Vanila JS, News Api, Word cloud.
+_description_ This is a Privacy card game to educate internet newbies about CyberSecurity in an interactive way. This is a card game that takes the player through different privacy situations and ask for the response they will choose. The points given to them specifies how good their choice is.
+
+_stack_ Made with JavaScript. 
+
+_hosted link_ https://hacket.netlify.com/
+
+#### Tracc - Personal Expense Tracker
+
+_description_ Tracc is a expense tracker build with React. It uses functional components with hooks and the context API.
+
+_stack_ Made with React. 
+
+_hosted link_ https://tracc.netlify.com/
 
 ### blogs
 
 I contribute my blogs to different websites. Some of them can be found at - https://www.csestack.org/author/pathak/
+
+## Blog 1
```

Comment:
Instead of Blog 1 add the title of your blog here and in the _description_ describe your blog in 2-3 lines.

Reply:
Resolved it. Please check.